symengine/tests/eval/test_eval_double.cpp

```diff
@@ -107,6 +107,8 @@ TEST_CASE("eval_double: eval_double", "[eval_double]")
         {SymEngine::atan2(r1, neg(r2)), 2.08867384922582},
         {mul(pi, mul(E, EulerGamma)), 4.92926836742289},
         {pow(mul(EulerGamma, r4), integer(8)), 4813.54354505117582},
+        {pow(E, real_double(0.2)), 1.22140275816017},
```

Comment:
See this test, https://github.com/symengine/symengine/blob/master/symengine/tests/basic/test_arit.cpp#L761-L769
You can have a new test over there.